ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
```

Comment:
Please remove the spaces around the equals signs. When specifying default values for keyword parameters, those are just not added in Python (including other methods in this project).

Reply:
Ok, changed here... But there are equal signs surrounded by spaces almost everywhere in the ibmdbpy package and it still works (?)... starting from this class `_init_` for example: `def __init__(self, idadb, tablename, indexer = None):` so I am surprised about this comment.

Reply:
The number of spaces will indeed not stop the code from working, it's more of a convention to not add spaces all over the place. The ibmdbpy code is unfortunately not really consistent in the style it uses.

Reply:
ok I understand.
I have removed the spaces around the equal signs in my own code but unfortunately I do not have the time to do it everywhere in the package.

Reply:
Note that spaces around equals signs are only not used when dealing with keyword parameters (either in `def foo(a=2)` or `foo(a=2)`. For normal assigments, they're still used. It's not a big deal, though, you don't have to go through the code and change them :-)